src/Signature/SignatureV4.php

```diff
@@ -67,19 +67,26 @@ public function signRequest(
     public function presign(
         RequestInterface $request,
         CredentialsInterface $credentials,
-        $expires
+        $expires,
+        array $options = []
     ) {
+        if (isset($options['start_time'])) {
```

Comment:
Optional: conditional assignments like this are clearer as ternary expressions than as a series of statements.